### PR TITLE
translated user_integration and positive_environmenal_impact

### PR DESCRIPTION
**Pull request by expeditionengineer**

### CODE REVIEW

File: run [hunk 1]
```diff
@@ -21,20 +21,22 @@ else
...
 
-HEADLESS=1
+HEADLESS=${HEADLESS}
+
 # Set Docker container ID
 container_id=webcentral
 currentDirInContainer=$(docker exec $container_id pwd)
+
+print_usage() {
+  printf "run script: -v <command>"
```
Comment (c0nb4) [inline run:31]: For what do we need the print statement?

**Comment by c0nb4** [comment]
Code looks good, have to check text on website. But cannot build locally due the following error: 

Error response from daemon: Conflict. The container name "/database" is already in use by container "ad67b546c357fd9328c63815c7b5ef8bd69d86a77ddfe3f43773b7b3f804beb4". You have to remove (or rename) that container to be able to reuse that name. 

I know this is not related to the code in this issue, but to the general previous refactoring. Would like to confirm or deny the reason for this. Tested with `./run up dev`

File: webcentral/test/06_system_test/test/scripts/test_user_integration.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+from src.page_obj.pages.comparison_page_section import ComparisonPageSection
+from src.page_obj.pages.user_integration_page import UserIntegrationPage
+from src.page_obj.pages.overview_page_section import OverviewPageSection
+from src.page_obj.pages.details_page import DetailsPage
+
+
+class TestUserIntegration(WebDriverSetup):
+    """Represent the Selenium-Test of the Components-List Page"""
+
+    # def testLinksToUserEngagementSite(self):
```
Comment (c0nb4) [inline webcentral/test/06_system_test/test/scripts/test_user_integration.py:31]: Can this be deleted?

File: webcentral/test/06_system_test/test/scripts/test_user_integration.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    #                     or "Nutzendenintegration" in self.driver.title)
+    #     self.driver.back()
+    #     userEngagementLinks = navbar.returnUserEngagementLink()
+    #     navbarActivateDropdownButton = navbar.getOperationalDropdownButton()
+    #     navbarActivateDropdownButton.click()
+    #     userEngagementLinks[0].click()
+    #     self.assertTrue("User integration" in self.driver.title
+    #                     or "Nutzendenintegration" in self.driver.title)
+    #     self.driver.back()
+    #
```
Comment (c0nb4) [inline webcentral/test/06_system_test/test/scripts/test_user_integration.py:66]: Until here

File: webcentral/test/06_system_test/testrunner.py
Comment (c0nb4) [inline webcentral/test/06_system_test/testrunner.py:1]: Please delete unused import statements

File: webcentral/test/08_test_data/test_criteria_catalog.xlsx
Comment (c0nb4) [inline webcentral/test/08_test_data/test_criteria_catalog.xlsx:1]: Why is the criteria catalog changed in this branch?

File: webcentral/test/06_system_test/testrunner.py
Comment (c0nb4) [inline webcentral/test/06_system_test/testrunner.py:1]: Please delete unused import statements. (The ones that start with # )

File: webcentral/test/06_system_test/src/page_obj/pages/details_page.py
Comment (c0nb4) [inline webcentral/test/06_system_test/src/page_obj/pages/details_page.py:1]: Why is this file empty?

**Review by c0nb4 — CHANGES_REQUESTED** [review]
Cannot test and verify the build due to: 


http://localhost:8000/de/pages/userEngagement 

<html><body>
<h1>NameError
       at /de/pages/userEngagement</h1>
  <pre class="exception_value">name 'explanationTextParagraphOne' is not defined</pre>
  
Request Method: | GET
-- | --
http://localhost:8000/de/pages/userEngagement
4.2.1
NameError
name 'explanationTextParagraphOne' is not defined
/webcentral/src/pages/views.py, line 118, in userEngagement
pages.views.userEngagement
/usr/local/bin/python3
3.10.14
['/webcentral/src',  '/usr/local/lib/python310.zip',  '/usr/local/lib/python3.10',  '/usr/local/lib/python3.10/lib-dynload',  '/usr/local/lib/python3.10/site-packages']
Thu, 29 Aug 2024 13:28:05 +0000


</body>
</html>

File: run (hunk 1 above)
Comment (expeditionengineer) [reply run:31]: I wanted to add a optional flag to make it possible to run the selenium tests in headless mode, but it doesnt work at the moment. I can remove the code.

File: webcentral/test/06_system_test/testrunner.py
Comment (expeditionengineer) [reply webcentral/test/06_system_test/testrunner.py:1]: sorry these statements are still needed. Im in the process of updating the filenames to snake_case

**Comment by c0nb4** [comment]
Fixed migrations, lgtm.

**Review by c0nb4 — APPROVED** [review]

**Review by c0nb4 — COMMENTED** [review]
LGTM.

**Pull request merged**